docs/src/docs/deploy.md

```diff
@@ -0,0 +1,112 @@
+
+## Pre-built Containers
+
+There are currently two pre-built containers:
+
+1. Init container: https://github.com/jozu-ai/kitops/blob/main/build/dockerfiles/init/README.md
+1. Kit CLI container: https://github.com/jozu-ai/kitops/blob/main/build/dockerfiles/README.md 
+
+## Init Container
+
+The init container unpacks the model reference from a ModelKit to a specific path and then exits. This makes it useful as a Kubernetes `init` container. This container also supports verifying signatures for containers automatically from key-based or keyless signers.
+
+The container is configurable via environment variables:
+
+`$MODELKIT_REF`: The ModelKit to pull (required).
+`$UNPACK_PATH`: Where to unpack the ModelKit (normally you’d want a `volumeMount` here). This is required and will default to `/home/user/modelkit`.
+`$UNPACK_FILTER`: Optional filter to limit what is unpacked (e.g., just the model, or model + code). The filter format is the same as the [–filter command line argument](./cli/cli-reference.md) for the Kit CLI.
+`$COSIGN_KEY`: Path to the key that should be used for verification, mounted inside the init container (e.g., from a Kubernetes secret).
+`$COSIGN_CERT_IDENTITY`: Signing identity for keyless signing.
+`$COSIGN_CERT_OIDC_ISSUER`: OIDC issuer for keyless signer identity.
+
+### Example Kubernetes YAML
+
+```
+ apiVersion: v1
+ kind: Pod
+ metadata:
+   name: my-modelkit-pod
+ spec:
+   containers:
+     - name: model-server
+       image: "" # Some container that expects your modelkit
+       # Share a volumeMount between the init container and this one
+       volumeMounts:
+         - name: modelkit-storage
+           mountPath: /my-modelkit
+
+   # Run the init container to unpack the ModelKit into the volume mount and make
+   # it available to the main container
+   initContainers:
+     - name: kitops-init
+       image: ghcr.io/jozu-ai/kitops-init-container:latest
+       env:
+         - name: MODELKIT_REF
+           value: "ghcr.io/jozu-ai/my-modelkit:latest"
+         - name: UNPACK_PATH
+           value: /tmp/my-modelkit
+       volumeMounts:
+         - name: modelkit-storage
+           mountPath: /tmp/my-modelkit
+
+   # Define a volume to store the ModelKit
+   volumes:
+     - name: modelkit-storage
+       emptyDir: {}       
+```
+
+## Using the Kit CLI Container
+
+The containerized Kit CLI can be used to tailor the running of a ModelKit because you can run any Kit CLI command. This gives you flexibility, but more manual work (the world is your oyster, but it may be hard to shuck).
+
+This container runs `kit` as its entrypoint, accepting Kit CLI arguments. You can also replace any time you use a kit CLI command with `docker run ghcr.io/jozu-ai/kit:latest` and it will run Kit inside the container. This means if you want to download it to your local machine, you’ll need to mount a docker volume, but you can use this to inspect ModelKits without installing the CLI.
```

Comment:
We can maybe trim out the "replace calling the Kit CLI bit" from this paragraph -- saying "you can run the container instead of the Kit CLI" is a useful description but I worry it will confuse people if they actually try it (e.g. "I ran kit unpack using the docker command and nothing showed up!"

I think I meant this more as a conceptual "think about the container like you think about the CLI" rather than an actual "try it", since the actual try it means getting into docker's semi-arcane volume mounting system.

Reply:
@amisevsk please check the updated wording.